klog/app/bookmark.go

```diff
@@ -213,3 +214,7 @@ func (bc *bookmarksCollection) ToJson() string {
 	}
 	return buffer.String()
 }
+
+func getDefaultBookmarkName() Name {
```

Comment:
I think this would cause a regression, since there doesn’t seem to be a fallback to `BOOKMARK_DEFAULT_NAME` (i.e., `default`) in case the `KLOG_DEFAULT_BOOKMARK` environment variable is not set.

Reply:
There is. The change is backwards compatible. We pass the value to `NewName` that will ensure the default is chosen if the value is empty.